### ISSUE TITLE
correctly interpret newline character also if they're written in uppercase (\N)

### ISSUE THREAD

**Issue by j-ed** [nextcloud/contacts#393]
## Expected behavior
Newline characters are usually written/saved as `\n`. Due to the fact that some applications seem to convert newline characters to uppercase, the contacts app should be tolerant enough to interpret it correctly.

## Current behavior
If newline characters are written in uppercase (\N) for whatever reason they're currently not interpreted correctly and displayed as it.

![20171008_nc_contacts_newline_in_comment](https://user-images.githubusercontent.com/2572764/31316829-96dfb004-ac35-11e7-9907-2d698564bcab.png)

A vcard export which is incorrectly interpreted looks like this:
```
NOTE:leckerer Kuchen!\N\NÃ–ffnungszeiten:\NMi-So: 14:00-18:30h
```

A vcard export which is correctly interpreted by the contacts app looks like this:
```
NOTE:leckerer Kuchen!\n\nÃ–ffnungszeiten:\nMi-So: 14:00-18:30h
```

### Steps to reproduce 
1. Sync contact data with e.g. Android/iPhone devices. I've not yet identified which of them are using uppercase newline characters.

#### Server Configuration
OS: Linux 3.16.47
Web server: Apache2 2.4.26
Database: MariaDB 10.0.32
PHP version: 5.6.30
Nextcloud version: 12.03
Contacts app version: nightly 2017-10-07

#### Client Configuration
Browser: Mozilla Firefox 56.0
Operating system: Windows 7

**Comment by irgendwie** [comment]
Sounds like a bug in the vcard lib we are using! Will take a look

**Comment by skjnldsv** [comment]
Fixed in 3.0.0
